landing-page/components/Header.tsx

```diff
@@ -1,18 +1,47 @@
 import { Span } from '@helpwave/common/components/Span'
 import { Helpwave } from '@helpwave/common/icons/Helpwave'
 import { tw } from '@helpwave/common/twind'
-import { Menu, X } from 'lucide-react'
-import Link from 'next/link'
+import { X, Menu as MenuIcon } from 'lucide-react'
 import { useState } from 'react'
+import Link from 'next/link'
+import { Menu, MenuItem } from '@helpwave/common/components/user-input/Menu'
 
-const homeURL = '/'
+const homeURL: string = '/'
+
+type LinkType = {
+  name: string,
+  url: string
+}
 
-const items = [
-  { name: 'tasks', url: '/product/tasks' },
-  { name: 'story', url: '/story' },
-  { name: 'team', url: '/team' },
-  { name: 'talks', url: '/talks' },
-  { name: 'contact', url: 'mailto:contact@helpwave.de' },
+type SubLinkType = LinkType & {
+  subpage?: LinkType[]
+}
+
+const items: SubLinkType[] = [
+  {
+    name: 'product',
+    url: '/product',
```

Comment:
```suggestion
    url: '/products',
```

Not sure if there will be a subpage on `/products`...

Reply:
Maybe this button should not be clickable as a Link.